.envrc

```diff
@@ -70,6 +70,7 @@ export PGUSER=postgres
 export PGPASS=mysecretpassword
 export PGPASSWORD=mysecretpassword
 export PGSSLMODE=disable
+export DB_MAX_CONNECTIONS=20
```

Comment:
I _think_ the way that this works is the default will be 20, but if we set an env variable in chamber, it will override this value when the container is built. Is that correct?

Reply:
I'm... not sure. I thought the value here was only used locally, or maybe in CI tests